bc_obps/reporting/service/report_non_attributable_service.py

```diff
@@ -0,0 +1,30 @@
+from typing import List
+
+from reporting.models import ReportNonAttributableEmissions, GasType, EmissionCategory
+
+from reporting.schema.report_non_attributable_emissions import ReportNonAttributableIn
+
+
+class ReportNonAttributableService:
+    @classmethod
+    def get_report_non_attributable_by_version_id(cls, report_version_id: int) -> List[ReportNonAttributableEmissions]:
+        return list(ReportNonAttributableEmissions.objects.filter(report_version__id=report_version_id))
+
+    @classmethod
+    def save_report_non_attributable_emissions(
+        cls, version_id: int, data: ReportNonAttributableIn
+    ) -> ReportNonAttributableEmissions:
+        emission_category = EmissionCategory.objects.get(category_name=data.emission_category)
+
+        report_non_attributable = ReportNonAttributableEmissions.objects.create(
```

Comment:
Isn't this going to create a new record even if a record is being edited? Should something like `update_or_create..` with the id of the record be used here instead?